charts/karpenter-crd/templates/karpenter.k8s.aws_ec2nodeclasses.yaml

```diff
@@ -95,6 +95,8 @@ spec:
                             rule: self.matches('^[a-zA-Z0-9]*@.*$')
                           - message: 'family is not supported, must be one of the following: ''al2'', ''al2023'', ''bottlerocket'', ''windows2019'', ''windows2022'''
                             rule: self.find('^[^@]+') in ['al2','al2023','bottlerocket','windows2019','windows2022']
+                          - message: windows families may only specify version 'latest'
```

Comment:
Is this guaranteed to have something after `@`. If I look at the regex above, I can see that I could end the value with `@` which I think would cause this validation to fail with a panic

Reply:
Looks like a mistake in that regex, it really should be `^[a-zA-Z0-9]+@.+$`. I'll update.